airlock/business_logic.py

```diff
@@ -41,6 +41,11 @@ class RequestFileType(Enum):
     SUPPORTING = "supporting"
 
 
+class FileApprovalStatus(Enum):
```

Comment:
minor naming suggestion

```suggestion
class FileReviewStatus(Enum):
```

This reads a bit better to me? Makes the type of `FileReview.status` to be `FileReviewStatus`. 

Could even just be ReviewStatus?

Reply:
yeah fair point, going with `FileReviewStatus`.